### PR TITLE
Add net.bartkessels.SimpleConvert

### PR DESCRIPTION
**Pull request by bartkessels**
[SimpleConvert](https://github.com/bartkessels/simpleconvert) is a small application to convert video and/or audio files to any filetype using ffmpeg. This application aims to provide an easy to understand interface so everyone will be able to use the power of ffmpeg.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build net.bartkessels.SimpleConver

**Comment by nedrichards** [comment]
bot, build net.bartkessels.SimpleConvert

File: net.bartkessels.simpleconvert.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	"app-id": "net.bartkessels.SimpleConvert",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.9",
+	"sdk": "org.kde.Sdk",
+	"command": "simpleconvert",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
+		"--filesystem=host"
```
Comment (nedrichards) [inline net.bartkessels.simpleconvert.json]: what's the problem with just using the portal, folders?

**Comment by nedrichards** [comment]
The [appdata](https://github.com/bartkessels/simpleconvert/blob/master/resources/linux/net.bartkessels.simpleconvert.appdata.xml) also requires [OARS](https://hughsie.github.io/oars/) tags.

**Comment by nedrichards** [comment]
The JSON file needs the same capitlization as the app-id. so not `net.bartkessels.simpleconvert.json` but `net.bartkessels.SimpleConvert.json`

File: net.bartkessels.simpleconvert.json (hunk 1 above)
Comment (bartkessels) [reply net.bartkessels.simpleconvert.json]: The filesystem=host permission is needed for the ffmpeg process to write the input file(s) to the selected location.

**Comment by bartkessels** [comment]
The OARS tags are added to the appdata.xml file and the json file has been renamed to match the app-id.

**Comment by nedrichards** [comment]
bot, build net.bartkessels.SimpleConvert

**Comment by nedrichards** [comment]
Build fails: https://flathub.org/builds/#/builders/2/builds/2820 I believe this is because:

> Not exporting share/applications/net.bartkessels.simpleconvert.desktop, non-allowed export filename

Everything has to have that capitalization.

**Comment by nedrichards** [comment]
> Also if data files already exist with the lower case version, use that for the package.

yes exactly.

**Comment by barthalion** [comment]
I'm closing it due to inactivity – but please re-open it or resubmit as new pull request if you would like to see it merged in the future.

**Pull request closed without merging**